airbyte-integrations/connectors/destination-customer-io/src/test-integration/kotlin/CustomerIoWriterTest.kt

```diff
@@ -54,7 +54,7 @@ object CustomerIoDataCleaner : DestinationCleaner {
 class CustomerIoWriterTest :
     BasicFunctionalityIntegrationTest(
         configContents = Files.readString(Path.of("secrets/config.json")),
-        configSpecClass = CustomerIoSpecification::class.java,
+        configSpecClass = NoopSpec::class.java,
```

Comment:
The spec is passed here only for `dumpAndDiffRecords` which we don't use for Data Activation destinations so I think we should be good here